test/usage.js

```diff
@@ -874,6 +877,52 @@ describe('usage tests', function () {
       r.logs[0].should.eql('1.0.1')
     })
 
+    describe('default version #', function () {
+      before(function (done) {
+        this.timeout(10000)
+        // create a symlinked, and a physical copy of yargs in
+        // our fixtures directory, so that we can test that the
+        // nearest package.json is appropriately loaded.
+        cpr('./', './test/fixtures/yargs', {
+          filter: /node_modules|example|test|package\.json/
```

Comment:
I was thinking the same, but opted to be lazy -- I'll update the test accordingly.